app/src/main/res/layout/activity_birthday.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.hedvig.app.feature.birthday.BirthdayActivity">
+
+    <TextView
+        style="@style/Header"
```

Comment:
If possible, we should use one of the entirely qualified styles and not set any `textSize` directly on the `TextView`. `@style/Header.Large` seems to match the style you're looking for here 🙂